.github/checklist.yml

```diff
@@ -0,0 +1,5 @@
+paths:
+  "lib/**.pm":
+    - Consider adding or extending unit tests in t/
```

Comment:
I like where your heart is at, not sure if it's the  right way though.

Reply:
Well, it says "consider", not "you *must*" :)

Reply:
> Well, it says "consider", not "you _must_" :)

That's the le's experiment part